doc/transform_methods/rbasex-math.rst

```diff
@@ -0,0 +1,810 @@
+regularization is not always valid for multiphoton processes, it can be useful
+in some cases and is easy to implement. To remind that it is not “truly
+non-negative”, this regularization is called “positive” here.
+
+
+A general advice applicable to all regularization methods is that when a
+relevant model is available, it is better to fit it directly to non-regularized
+results, thus avoiding additional assumptions and biases introduced by
+regularizations.
+
+
+Examples
+^^^^^^^^
+
+.. warning::
+    Absolute and relative efficiencies of these regularization methods and
+    their optimal parameters depend on the image size, the amount of noise and
+    the distribution itself. Therefore *do not assume* that the examples shown
+    here are directly relevant to *your* data.
+
+Some properties of the regularization methods described above are demonstrated
+here by applying them to a synthetic example. The test distribution from the
+BASEX article is forward Abel-transformed to obtain its projection, and then
+Poissonian noise is added to it to simulate experimental VMI data with
+relatively low signal levels (such that the noise is prominent):
+
+.. comment
+    the only purpose of ":scale: 1" in the plots below is to make them clickable
+
+.. plot:: transform_methods/rbasex-sim.py
+    :scale: 1
+
+In order to characterize the regularization performance, all the methods are
+applied at various strengths to this simulated projection, and the relative
+root-mean-square error :math:`\big\|\tilde I(r) - I_\text{src}(r)\big\| \big/
+\big\|I_\text{src}(r)\big\|`, where :math:`I_\text{src}(r)` is the “true”
+radial intensity distribution, and :math:`\tilde I(r)` is the reconstructed
+distribution, is calculated in each case. The following plot shows how this
+reconstruction error changes with the regularization strength (the
+non-parameterized “pos” method is shown by a dashed line):
+
+.. plot:: transform_methods/rbasex-regRMSE.py
+    :align: center
+
+(Note that the horizontal axis in the left plot is nonlinear, and that the
+vertical axis in the right plot does not start at zero and actually spans a
+very small range.)
+
+These plots demonstrate that the Tikhonov methods have some optimal strength
+value, at which the reconstruction error is minimized. At smaller values the
+noise is not suppressed enough (zero strength corresponds to the
+non-regularized transform), and at larger values the reconstructed distribution
+is smoothed too much.
+
+The SVD plot has steps corresponding to successive removal of singular values.
+The reconstruction error does not decrease monotonically, but exhibits several
+local minima before starting to grow. Notice that even the global minimum is
+only slightly better than no regularization.
+
+The actual reconstructed images for each regularization method at its optimal
+strength are shown below with their radial intensity distributions:
+
+
+Using ``reg=None``
+""""""""""""""""""
```

Comment:
This will be difficult. The tick labels already overlap with the image to save space. This is not very noticeable, but the vertical-axis label will be drawn over the image and look very bad. Making the panels separated will squeeze the plots too much, and the whole figure cannot be made wider. Maybe this is not a big deal, since everything is explained immediately below the figure?

Reply:
okay, we can skip the labels.